docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -39,342 +37,123 @@ Devices with Intel® GNA support:
-
-## Intel® GNA Forward and Backward Compatibility
+### Intel® GNA Forward and Backward Compatibility
 
 In the general case, there is no guarantee that a model compiled for GNA 2.0 will run on GNA 3.0, or vice versa.
 
-However, in most cases, networks compiled for GNA 2.0 will run as expected on GNA 3.0, although the performance may be worse compared to the case when a network is compiled specifically for the latter.  The exception is networks with convolutions with the number of filters greater than 8192 (see the <a href="#models-and-layers-limitations">Models and Layers Limitations</a> section).
+However, in most cases, networks compiled for GNA 2.0 will run as expected on GNA 3.0, although the performance may be worse compared to the case when a network is compiled specifically for the latter.  The exception is networks with convolutions with the number of filters greater than 8192 (see the <a href="#models-and-operations-limitations">Models and Operations Limitations</a> section).
 
 Networks compiled for GNA 3.0 should run on GNA 2.0 with incompatible layers emulated on CPU.
 
-You can use the following options `KEY_GNA_EXEC_TARGET` and `KEY_GNA_COMPILE_TARGET` options  to check interoperability (see the <a href="#supported-configuration-parameters">Supported Configuration Parameters</a> section below):
+You can use the following options `ov::intel_gna::execution_target` and `ov::intel_gna::compile_target` properties to check interoperability.
 
-@sphinxdirective
-.. tab:: C++
+## Software emulation mode
 
-   ``KEY_GNA_EXEC_TARGET``,  ``KEY_GNA_COMPILE_TARGET``
-
-.. tab:: Python
+On platforms without GNA hardware support plugin chooses software emulation mode automatically by default.
+GNA plugin enables you to switch the execution between software emulation mode and hardware execution mode after the model is loaded.
+For details, see using `ov::intel_gna::execution_mode` property.
 
-   ``GNA_EXEC_TARGET``,  ``GNA_COMPILE_TARGET``
+## Recovery from Interruption by High-Priority Windows Audio Processes\*
 
-@endsphinxdirective
+GNA is designed for real-time workloads such as noise reduction.
+For such workloads, processing should be time constrained, otherwise extra delays may cause undesired effects such as
+*audio glitches*. To make sure that processing can satisfy real-time requirements, the GNA driver provides a Quality of Service
+(QoS) mechanism, which interrupts requests that might cause high-priority Windows audio processes to miss
+the schedule, thereby causing long running GNA tasks to terminate early.
 
-## Drivers and Dependencies
+Applications should be prepared for this situation.
```

Comment:
Maybe just say how? Like we recommend to use Automatic QoS feature or something like that....